docs/api/ref/schemas/tags/tags-write.yaml

```diff
@@ -0,0 +1,27 @@
+type: object
+title: Tags (WRITE)
+description: |
+  Products have list of tags of different types (e.g. categories, labels) that can be replaced or added to with the following properties in API v3.
+properties:
+  categories_tags:
+    type: array
+    description: An array of categories tag entries that will replace existing categories
+    items:
+      $ref: ./input_taxonomy_tag_entry.yaml
+  categories_tags_add:
+    type: array
+    description: An array of categories tag entries that will be added to existing categories
+    items:
+      $ref: ./input_taxonomy_tag_entry.yaml
+
+patternProperties:
+  categories_tags_(?<language_code>\w\w):
```

Comment:
I added support for things like categories_fr_tags = "Gâteaux", but it means we need patternProperties. Maybe we could deprecate that feature. The alternative is to send tags_lc=fr + categories_tags = "Gâteaux", which is not very complex either.

Reply:
Yes would be better to show it as deprecated.